### PR TITLE
Style sort mathcing opprtunities

### PR DESCRIPTION
**Pull request by youssef910**

### CODE REVIEW

File: client/src/components/MatchingApplicantsAndOppotunities.js [hunk 1]
```diff
@@ -47,57 +47,81 @@ class MatchingApplicantsAndOppotunities extends Component {
...
+                            src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRePtynQytlAevO-zkIk1hJHVWGr0LmbNhm7MSbUIz63g3puHya"
+                            style={{ width: "10" }}
+                          ></Image>
+                        ) : null} */}
+                        <div style={{ fontSize: 12, marginTop: -5 }}></div>
+                      </CircularProgressbarWithChildren>
+                    </Card.Content>
+                  </Card.Content>
+                </Card>
+                <br />;
```
Comment (DwinaTech) [inline client/src/components/MatchingApplicantsAndOppotunities.js]: Remove semicolon.

File: client/src/components/MatchingApplicantsAndOppotunities.js [hunk 2]
```diff
@@ -47,57 +47,81 @@ class MatchingApplicantsAndOppotunities extends Component {
...
+                    </Card.Header>
+                    <Card.Description>
+                      {opportunity.description}
+                    </Card.Description>
+                    <Card.Content textAlign="right" style={{ width: "100px" }}>
+                      <CircularProgressbarWithChildren
+                        value={opportunity.percentage}
+                        text={`${opportunity.percentage}%`}
+                      >
+                        {/* {opportunity.percentage > 60 ? (
```
Comment (DwinaTech) [inline client/src/components/MatchingApplicantsAndOppotunities.js]: Remove the comment out code if you do not need it.

File: client/src/components/MatchingApplicantsAndOppotunities.js [hunk 3]
```diff
@@ -47,57 +47,81 @@ class MatchingApplicantsAndOppotunities extends Component {
...
-                  styles={buildStyles({
-                    strokeLinecap: "butt"
-                  })}
+      <Grid stackable>
+        <Grid.Row columns={3}>
+          {matchingOpportunities.map(opportunity => {
+            return (
+              <Grid.Column>
+                {" "}
+                <Card
```
Comment (DwinaTech) [inline client/src/components/MatchingApplicantsAndOppotunities.js]: Pass `fluid` property to Card to take the full width on a small screen.

File: client/src/components/MatchingApplicantsAndOppotunities.js (hunk 2 above)
Comment (youssef910) [reply client/src/components/MatchingApplicantsAndOppotunities.js]: Done

File: client/src/components/MatchingApplicantsAndOppotunities.js (hunk 1 above)
Comment (youssef910) [reply client/src/components/MatchingApplicantsAndOppotunities.js]: Done

File: client/src/components/MatchingApplicantsAndOppotunities.js (hunk 3 above)
Comment (youssef910) [reply client/src/components/MatchingApplicantsAndOppotunities.js]: Done

**Pull request merged**